### DISCUSSION TITLE
Comment with your ideas for new scripts or features additions

### DISCUSSION THREAD

**Discussion by 007revad** [007revad/todo_list#2] [Ideas]
Comment with your ideas for new scripts or features to add to my existing scripts.

[Synology_HDD_db](https://github.com/007revad/Synology_HDD_db)
[Synology_M2_volume](https://github.com/007revad/Synology_M2_volume)
[Synology_enable_M2_volume](https://github.com/007revad/Synology_enable_M2_volume)
[Plex_Server_Sync](https://github.com/007revad/Plex_Server_Sync)
[Synology_RAID-F1_SHR_switch](https://github.com/007revad/Synology_RAID-F1_SHR_switch)
[Synology_Config_Backup](https://github.com/007revad/Synology_Config_Backup)
[Synology_Plex_Backup](https://github.com/007revad/Synology_Plex_Backup)
[Synology_enable_Deduplication](https://github.com/007revad/Synology_enable_Deduplication)
[ScriptNotify](https://github.com/007revad/ScriptNotify)

[Asustor_Plex_Backup](https://github.com/007revad/Asustor_Plex_Backup)

**Comment by vuolter** [comment]
Heey man, thanks for your amazing efforts (and ofc your scripts!), 

I wonder if the fan speed settings could be tweaked a bit to enable the "low-power mode" on other models besides the DS223j and DS224 (which have it by design).
I would like the fan to stop completely instead of staying at a lower speed all the time...
What do you think?

**Reply by 007revad** [reply]
I just compared the DS223 to the DS220+ to see what was different and surprisingly the DS220+ can run it's fans slower than the DS223 
- DS220+ fan speeds range from 20% to 99%
- DS223 fan speeds range from 50% to 99%

I'll have a look at the DS223j and DS224+ later.

**Reply by vuolter** [reply]
Hi @007revad, did you have time to investigate further? thx

**Reply by 007revad** [reply]
Sorry, no. I've been busy. Can you remind in a week if you haven't heard back from me.

**Reply by vuolter** [reply]
Heey @007revad np, let me know when you have some time. 😉

**Reply by 007revad** [reply]
What Synology NAS do you have?

I've just looked at the DS124's /usr/syno/etc.defaults/scemd.xml and it has a DUAL_MODE_LOW_STOP section which includes a 0% fan speed when the disk temp is below 40 degrees C and CPU is below 55 degrees C.
